draft-ietf-rtcweb-jsep.xml

```diff
@@ -1095,6 +1095,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           described in
           <xref target="sec.addTransceiver" />.</t>
         </section>
+        <section title="removeTrack" anchor="sec.removeTrack">
+
+          <t>The removeTrack method remove a MediaStreamTrack from the
+          PeerConnection, using the the sender argument to indicate which
```

Comment:
for addTrack, we used the type (i.e., "MediaStreamTrack") rather than the name of the formal argument.